src/world/action/land.c

```diff
@@ -1,9 +1,220 @@
 #include "common.h"
+#include "functions.h"
 
-INCLUDE_ASM(void, "world/action/land", func_802B6000_E24920, void);
+void func_802B62CC_E24BEC(void);
+void func_802B644C_E24D6C(void);
 
-INCLUDE_ASM(void, "world/action/land", func_802B61C0_E24AE0, void);
+void func_802B6000_E24920(void) {
+    f32 inputMoveMagnitude;
+    f32 inputMoveAngle;
+    s32 jumpInputCheck;
+    s32 phi_a0;
+    CollisionStatus* currentCollisionStatus = &gCollisionStatus;
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Camera* currentCameras = &gCameras[0];
```

Comment:
These 3 lines with initialized stuff should go first. Also "currentCameras" is actually just the "Default" camera (0 should instead be CAM_DEFAULT and the name should be just `camera`)